build-vm

```diff
@@ -1024,7 +1039,8 @@ vm_save_statistics() {
     test -e /dev/sda && device="sda"
     test -e /dev/vda && device="vda"
     test -e /dev/xvda && device="xvda"   # in newer XEN setups
-    test -e /dev/dasda && device="dasda" # in z/VM
+    # while there is dasda in z/VM, it can happen that dasda really is the swap device.
+    test -e /dev/dasda && device=$(ls /sys/bus/ccw/devices/0.0.${ZVM_VOLUME_ROOT}/block) # in z/VM
```

Comment:
why is the ls command needed here at all?

Reply:
When booting the worker guest, /dev/dasda or /dev/dasdb are not considered persitent devices. To have some kind of persistency, I use the (known) value of the SWAP device to determine the actual device name. Typically will also resolve to dasda, because it has the lower number.